app/actions/views/file_upload.js

```diff
@@ -18,6 +18,11 @@ export function handleUploadFiles(files, rootId) {
         const clientIds = [];
 
         files.forEach((file) => {
+            if (Platform.os !== 'ios') {
```

Comment:
can you be specific here like `Platform.OS ===  'android'` btw Platform.os is `undefined`

Reply:
oh. When I console.logged it, it was returning "undefined" on my android devise. I assumed (wrongly it seems) Platform.OS was a iOS only thing.

Reply:
Sorry, re-reading your comment I am more confused. "btw Platform.os is undefined"

^ Yeah, that was why I did `!== 'ios'`

Reply:
"also I think you missed adding this to the action and to the client driver in the mattermost-redux library" - does it need to be changed in the redux library? Perhaps someone may have to walk through this with me.

It currently seems to work with no redux updates, since the redux action simply passes on the headers

Reply:
no need to do anything on redux as is part of the formData